### PR TITLE
SCMOD-4072: Updated to use latest dropwizard

### PR DESCRIPTION
**Pull request by Anthony-McGreevy**
https://jira.autonomy.com/browse/SCMOD-4072

### CODE REVIEW

**Comment by Anthony-McGreevy** [comment]
**TEST BUILD:** http://sou-jenkins2.hpeswlab.net/job/CAFDataProcessing/view/Developer/job/CAFDataProcessing~worker-document~SCMOD-4072-testbuild~CI/

File: pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+    Copyright 2018-2017 EntIT Software LLC, a Micro Focus company.
```
Comment (dermot-hardy) [inline pom.xml]: Project inception year should remain 2015.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Comment by Anthony-McGreevy** [comment]
@dermot-hardy Inception year has been updated

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
- Version number needs rolled on
- Release notes need updated

**Comment by Anthony-McGreevy** [comment]
@dermot-hardy version and release notes updated.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**